sql/attribute.go

```diff
@@ -146,7 +151,7 @@ func (a *attribute) GenerateCode() (string, error) {
 		return fmt.Sprintf("%s=%s", a.Name,
 			strings.Join(strings.Split(fmt.Sprint(intList), " "), ",")), nil
 	}
-	return "", fmt.Errorf("value of attribute must be string or list of int, given %s", a.Value)
+	return "", fmt.Errorf("The Value type of an attribute must be string, int, float or list of ints, given %s", a.Value)
```

Comment:
Error string should not be capitalized: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Done.